lab_3_generate_by_ngrams/main.py

```diff
@@ -309,6 +450,21 @@ def get_next_token(self, sequence: tuple[int, ...]) -> Optional[list[tuple[int,
 
         In case of corrupt input arguments or methods used return None.
         """
+        if not(isinstance(sequence, tuple) and sequence):
+            return None
+        tokens = self._model.generate_next_token(sequence)
+        if tokens is None:
+            return None
+        if not tokens:
+            return []
```

Comment:
in lines 459 and 457 you are basically returning the `tokens`, right? consider shortening this check-return block

Reply:
I tried to change it but it didn't work out. As I understand we should return None if generate_next_token return None and if dictionary of next_tokens is empty it should return [ ]. Could you drop me a hint how to do this?